packages/theme/pages/Checkout/Shipping.vue

```diff
@@ -7,7 +7,7 @@
       class="sf-heading--left sf-heading--no-underline title"
     />
     <AddressPicker
-      v-if="isAuthenticated && savedAddresses"
+      v-if="isAuthenticated && savedAddresses && !isFormSubmitted"
```

Comment:
The v-if is getting complicated; could we transform it into isAdressPickerVisible `computable` to improve readability?